test/test_tensor_creation_ops.py

```diff
@@ -887,8 +887,8 @@ def test_arange_bfloat16(self, device):
         bfloat16_tensor = torch.arange(0, 6, step=2, dtype=torch.bfloat16, device=device)
         self.assertEqual(ref_tensor, bfloat16_tensor)
 
-    @dtypes(torch.int8, torch.short, torch.int, torch.long, torch.float, torch.double)
-    @dtypesIfCUDA(torch.int8, torch.short, torch.int, torch.long, torch.half, torch.float, torch.double)
+    @dtypes(*torch.testing.get_all_dtypes(include_bool=False, include_half=False))
+    @dtypesIfCUDA(*torch.testing.get_all_dtypes(include_bool=False, include_half=True))
```

Comment:
Should bfloat be included as well? And does `get_all_dtypes` return complex?

Reply:
Complex and bfloat16 are included by default

Reply:
Yes, you are right
```python
def get_all_dtypes(include_half=True, include_bfloat16=True, include_bool=True, include_complex=True) -> List[torch.dtype]:
    dtypes = get_all_int_dtypes() + get_all_fp_dtypes(include_half=include_half, include_bfloat16=include_bfloat16)
    if include_bool:
        dtypes.append(torch.bool)
    if include_complex:
        dtypes += get_all_complex_dtypes()
    return dtypes
```